railties/lib/rails/generators/rails/app/templates/config/initializers/debug.rb.tt

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+if Rails.env.local?
+  require "debug"
+
+  # See https://github.com/ruby/debug#configuration-list for more options.
+  DEBUGGER__::CONFIG[:no_repeat] = true
```

Comment:
I think this is the easiest way to configure `debug` without messing around with environment variables (ref: https://github.com/ruby/debug/issues/1098).